libfastjson.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: libfastjson
+  version: 1.2304.0
+  epoch: 0
+  description: a fast json library for C
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
```

Comment:
Oh, and when using our `autoconf` pipelines, we don't need to include either of these. We pull them in as part of the pipelines

Reply:
Done.  However, I did have to leave `libtool` in there, as the build fails otherwise.  Should we have our `autoconf` pipeline pull in `libtool` too?